### DISCUSSION TITLE
Renovate not running npm install?

### DISCUSSION THREAD

**Discussion by Tonyhliu** [renovatebot/renovate#15240] [Request Help] [answered]
From my understanding, the flow should be as follows : 

1. The repository has a package.json and package-lock.json with version 1.0.0 of a dependency
2. Renovate sees that version 1.1.0 is available
3. Renovate patches the package.json to change the dependency's version from 1.0.0 to 1.1.0
4. Renovate runs npm install to let npm update the package-lock.json
5. Renovate commits the package.json and package-lock.json
6. Renovate creates the PR

However, it doesn't seem like step #4 is working as expected for me as shown in the error below in my pipeline :

```npm ERR! `npm ci` can only install packages when your package.json and package-lock.json or npm-shrinkwrap.json are in sync. Please update your lock file with `npm install` before continuing.```

the PR is generated correctly with the package bump in the `package.json`, but any ideas as to why it's not running install to also include the `package-lock` in its commit?

**Comment by viceice** [comment, marked as answer]
if you're self-hosted renovate user, you're probably hit by
- https://github.com/renovatebot/renovate/issues/15211

**Reply by viceice** [reply]
PR on the way
- https://github.com/renovatebot/renovate/pull/15242
